src/core_modules/capture-core/components/D2Form/SectionDescriptionBox.types.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+export type PlainProps = {|
```

Comment:
Actually, I would have switched the name of PlainProps and Props. As in, PlainProps are props for SectionDescriptionBoxPlain and Props are props for the exported component.